### DISCUSSION TITLE
Mastodon iPad app

### DISCUSSION THREAD

**Discussion by b1keridingpinko** [mastodon/mastodon#19636] [Q&A]
Hey everyone, I'm experiencing an issue on the iPad app where it doesn't give me the "options" at the bottom of creating a new post - attach a file or picture for example and all that follows.  Has anyone experienced that and is there a solution or is this a bug that needs to be reported somehow?  Thanks

**Comment by jeremieb** [comment]
It's the same if you're opening the app from an Apple Silicon Mac, impossible to add attachments when creating a post.

🙏

**Comment by Frau11riede** [comment]
Same here, app is only working properly when using an iPhone. Maybe iOS 16 is the problem?

**Comment by supernovae** [comment]
Also see people reporting missing keyboard on iPad - but working fine on phone.

**Reply by jeremieb** [reply]
It might be because we're using physical keyboard and right now it's a keyboard toolbar for adding a picture...
